### PR TITLE
Implemented new features

### PR DESCRIPTION
**Pull request by Pranav0-0Aggarwal**
Implemented features like:
1. Retrieve all rooms data
2. Retrieve data of single room
3. Retrieve data of a user.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
 # Resonate-Backend
+
+With the rising popularity of social voice platforms such as Clubhouse and Twitter Spaces, it is high time for an Open Source alternative. A platform like this would not only enhance credibility within the open-source community, but also attract more users and foster growth. An engagement platform that is Open Source has the potential to drive significant traction and help establish a strong presence.
+
+## Installation
```
Comment (chandansgowda) [inline README.md:5]: Can we change it to any other phrase. Installation is not the right work I guess. Something like Setting Up The Environment ?

File: index.js
Comment (chandansgowda) [inline index.js:1]: Actually, retrieving room details will happen directly through the app. Also Firebase Firestore Structure has not yet been finalized. Please remove this file until that's done. You can join the discussion in the related issue posted on Resonate repo.

**Review by chandansgowda — CHANGES_REQUESTED** [review]
One more thing I would like to add is that the commit structure is not maintained as required. Refer to README in main repo. I would suggest you do this as of now - Drop this PR and create a new branch and just get the README file edited. Once the firebase structure is finalized, you can work on the endpoints.

File: README.md (hunk 1 above)
Comment (Pranav0-0Aggarwal) [reply README.md:5]: thanks for pointing out.

File: index.js
Comment (Pranav0-0Aggarwal) [reply index.js:1]: ok, got that, are there any other required routes that I can design?

**Comment by Aarush-Acharya** [comment]
@Pranav0-0Aggarwal, thank you for your contributions but the PR is too old and is out of context now

**Pull request closed without merging**